backend/tracim_backend/lib/mail_notifier/notifier.py

```diff
@@ -300,7 +299,13 @@ def notify_content_update(self, event_actor_id: int, event_content_id: int) -> N
             # contains only message_id from parents post in thread.
             # To link this email to a content we create a virtual parent
             # in reference who contain the content_id.
-            message["References"] = formataddr(("", reference_addr))
+            references = formataddr(("", reference_addr))
```

Comment:
You probably can remove the use of `formataddr`. From [its documentation ](https://docs.python.org/3/library/email.utils.html#email.utils.formataddr): "If the first element of pair is false, then the second element is returned unmodified."

Reply:
modified in EmailAdress.adress property (after pulling develop code).